bleach/__init__.py

```diff
@@ -43,6 +44,8 @@
 
 ALLOWED_STYLES = []
 
+ALLOWED_PROTOCOLS = copy.copy(HTMLSanitizer.acceptable_protocols)
```

Comment:
@EmilStenstrom Which protocols? Where would you need this "BACKWARDS INCOMPATIBLE" warning? Is a note in the CHANGES file enough?